src/strategies/sound-protocol-get-tokenuri/transformer.mjs

```diff
@@ -0,0 +1,61 @@
+// @format
+import { resolve } from "path";
+import { env } from "process";
+
+import logger from "../../logger.mjs";
+
+export const name = "sound-protocol-get-tokenuri";
+const log = logger(name);
+export const version = "0.1.0";
+
+export function onClose() {
+  log("closed");
+  return;
+}
+
+export function onError(error) {
+  log(error.toString());
+  throw error;
+}
+
+export function onLine(line) {
+  const obj = JSON.parse(line);
+  const datum = obj.results;
+  return JSON.stringify({
+    version,
+    title: datum.name,
+    artist: {
+      version,
+      name: datum.artist_name,
+    },
+    platform: {
+      version,
```

Comment:
We should check what version we're assigning to the regular sound extraction strategy and if we're e.g. assigning "1.0.0", then here we should assign "2.0.0"